admin-manual/installation/installation.rst

```diff
@@ -75,35 +73,42 @@ however, processing large collections will require better hardware. See
 **Dependencies**
 
 Archivematica has a long list of software it depends on.  All of these
-dependencies are intalled when following the instructions below.
+dependencies are installed when following the instructions below.
 
 In these instructions everything is installed into one machine.  It is possible
 to install some of the components on separate machines, to improve performance,
 such as:
 
 * MySQL
-* Elasticsearch
 * gearman
+* Elasticsearch (optional as of |release|, see below)
 
 Using additional machines requires some additional configuration.
 
 See :ref:`Advanced <advanced>`.
 
-**Notes**:
+.. note:: *Indexing*. Installing Elasticsearch to provide a search index is now
+   optional as of version |release|.  Installing Archivematica in a headless state,
```

Comment:
Why this is called "headless"? We still expect users to use the web interface, afaik.

Reply:
You're right. I believe that this "remove Elasticsearch" (a.k.a. "indexless") work is heading in the direction of being able to run Archivematica in a completely "headless" way, such that you could do everything via the API and never even run the dashboard GUI. But clearly we are not there yet, so maybe we should just call this "indexless" until such time as we really have headless functionality.